src/XRP/BinaryCoding.h

```diff
@@ -47,4 +56,10 @@ inline void encodeBytes(std::vector<uint8_t> bytes, std::vector<uint8_t>& data)
     data.insert(data.end(), bytes.begin(), bytes.end());
 }
 
+inline void encode0(int len, std::vector<uint8_t>& data) {
```

Comment:
Better naming than 0 suffix?

Reply:
It's not a 0 suffix. It's a literal 0. I changed the name to encode0s() to make it clearer.

Reply:
OK, thanks. Note that I don't yet see these changes, it's OK if it will come a bit later.